src/components/TeacherViews/LessonMedia/LessonMedia.view.tsx

```diff
@@ -0,0 +1,89 @@
+import { OakTertiaryButton } from "@oaknational/oak-components";
+
+import { resolveOakHref } from "@/common-lib/urls";
+import Box from "@/components/SharedComponents/Box";
+import MaxWidth from "@/components/SharedComponents/MaxWidth";
+import Breadcrumbs from "@/components/SharedComponents/Breadcrumbs";
+import {
+  getLessonOverviewBreadCrumb,
+  getBreadcrumbsForLessonPathway,
+  getCommonPathway,
+  getLessonMediaBreadCrumb,
+} from "@/components/TeacherComponents/helpers/lessonHelpers/lesson.helpers";
+import { LessonPathway } from "@/components/TeacherComponents/types/lesson.types";
+
+type BaseLessonMedia = {
+  lessonTitle: string;
+  lessonSlug: string;
+};
+
+type CanonicalLesson = BaseLessonMedia & {
+  pathways: LessonPathway[];
+};
+
+type NonCanonicalLesson = BaseLessonMedia & LessonPathway;
+
+type LessonMediaProps =
+  | {
+      isCanonical: true;
+      lesson: CanonicalLesson;
+    }
+  | {
+      isCanonical: false;
+      lesson: NonCanonicalLesson;
+    };
+
+export function LessonMedia(props: LessonMediaProps) {
+  const { isCanonical, lesson } = props;
+  const { lessonTitle, lessonSlug } = lesson;
+
+  const commonPathway = getCommonPathway(
+    props.isCanonical ? props.lesson.pathways : [props.lesson],
+  );
+
+  const { programmeSlug, unitSlug } = commonPathway;
+
+  return (
+    <Box $ph={[16, null]}>
```

Comment:
Is it possible to refactor with component lib components i.e. `OakBox`, `OakMaxWidth` etc